config/config.go

```diff
@@ -160,6 +160,12 @@ type NamedEntity interface {
 	SetName(name string)
 }
 
+// Validator defines the interface for the configuration validation.
+type Validator interface {
```

Comment:
```suggestion
type Validatable interface {
```

Reply:
Done, also for the moment make it unexported